### PR TITLE
Inputs & selection families- aligning large size to 42px

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary

Inputs & selection families- aligning large size to 42px
Work in progress

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by TalZa1810** [comment]
Currently not merging since it breaks `LabelledElement`:
https://wix-wix-style-react-pr-5457.surge.sh/?path=/story/components-api-internal--labelledelement

There's no zeplin so I'm not sure how to fix this. Should discuss with UX regarding this change effect on this component.

**Comment by TalZa1810** [comment]
@giladsegal Where can I find the Zeplin file of `LabeledElement` ?

File: src/Input/Input.scss [hunk 1]
```diff
@@ -8,7 +8,6 @@ $double-gap: 12px;
 .root {
   position: relative;
   display: flex;
-  align-items: center;
```
Comment (EgoziE) [inline src/Input/Input.scss:11]: Double check if it's not breaking anything

**Comment by TalZa1810** [comment]
Fails on visuals which are not related to the change (`FillPreview` & `MediaOverlay`). 
The change was approved by Ayelet.

**Pull request merged**